smallcloudai/refact-chat-js
# Entertain user about subchat

# Entertain user about subchat



## Description

When a tool with sub chats got called, it looked like the chat got frozen. So add visualizations to let the user know what is happening and that it is not frozen.

## Type of change




## How to Test

Make the chat call the locate tool, and see if the sub chats and attached files appear.

## Checklist
